docs/docs/use-cases/overview.md

```diff
@@ -1,24 +1,37 @@
-# Use Cases
+# Overview
 
-## Assisted Prescriptions
+## Assisted Prescription
 
-This use case shows how to build a custom analytics application with Qlik Core.
+This use case is good to check out if you want to explore the following aspects of Qlik Core:
 
-* It has a custom-built web UI.
-* The backend uses Qlik Core and serves multiple users with a single document used by Qlik Associative Engine.
-* Load from multiple users is managed by scheduling sessions to different engine instances using a least-load strategy.
-* The application is deployed on [Amzon Web Services](https://aws.amazon.com/).
-* Docker Swarm is used as container orchestrator.
+* Building a custom UI on top of Qlik Core, using [picasso.js](https://github.com/qlik-oss/picasso.js) and
+  [enigma.js](https://github.com/qlik-oss/enigma.js).
+* Building a backend using Qlik Core to serve multiple users with a single document used by
+  [Qlik Associative Engine](../services/qix-engine/introduction.md).
+* Managing load from multiple users by scheduling sessions to different engines using a least-load strategy.
+* Deploying a Qlik Core application on Amzon Web Services.
+* Running [Mira](../services/mira.md) with Docker Swarm as container orchestrator.
+* Building an athentication solution using GitHub as identity provider.
+* Using Redis for managing session cookies and internal JWTs to be consumed by Qlik Associative Engine.
```

Comment:
Maybe mention JWT in the point above and drop this? A bit too detailed to mention redis I think.